_maps/FortuneGateFG1/FortuneGate.yaml

```diff
@@ -23,7 +23,7 @@ mapIcon: FortuneGate
 music:
   download: 
     - https://nikkums.io/cswt/BGM/FortuneGateFG1.music.zip
-    - https://drive.google.com/u/0/uc?id=1ymrEPQ50mw4yOIQYiCgo4Z8VD3lH_x5j&export=download
+    - https://drive.google.com/file/d/1tdsS_aU5A7vuHzPmHs4eTZwQjT0uGnv9/view?usp=drive_link

```

Comment:
Hello! This Google Drive link works in the browser, but it doesn't work for directly downloading through tools like CSMM. So I'm going to look through your .yaml files and offer correctly-formatted links for each entry. Please update these in your files when you get a chance.

The properly-formatted download link for this file should be `https://drive.google.com/uc?export=download&id=1tdsS_aU5A7vuHzPmHs4eTZwQjT0uGnv9`

One more note for this file specifically: the music zip in Google Drive seems to have a space in its filename. Please rename it to remove the space, as those can behave unpredictably across different OSes. Thank you for all your hard work on these boards!